### PR TITLE
Fix EVS crash when using `video_embeds` inputs in Qwen2.5-VL

### PR DESCRIPTION
**Pull request by skyloevil**
### **Summary**

Fix a crash in `Qwen2.5-VL` when EVS (Efficient Video Sampling) is enabled and video inputs are provided through the `video_embeds` pathway. The regression occurred because `second_per_grid_ts` is only present in the pixel path (`pixel_values_videos`) but was assumed to always exist in `_postprocess_video_embeds_evs`.

This patch makes the `video_embeds` path compatible with EVS by normalizing handling of `second_per_grid_ts` and preventing a failure in the pruning flow.

---

### Before

Repro conditions:

* `enable_mm_embeds=True`
* `video_pruning_rate>0`
* modality: `video_embeds`
* multimodal limits configured (so pruning enabled)

Observed behavior (before):

* `_postprocess_video_embeds_evs` is invoked
* missing `second_per_grid_ts`
* crash in `.long()`
* no LLM output
image
---

### **Fix**

Ensure `second_per_grid_ts` is appropriately handled for `video_embeds` inputs, enabling EVS to run consistently across both pixel-based and embedding-based video paths.

After the fix, EVS pruning + mRoPE position augmentation works correctly for `video_embeds`. 
image

---

### **Validation**
```python
import torch
from vllm import LLM, SamplingParams

llm = LLM(
    model="Qwen/Qwen2.5-VL-3B-Instruct",
    enable_mm_embeds=True,
    video_pruning_rate=0.5,
    trust_remote_code=True,
    dtype="bfloat16",
    max_model_len=10240,
)

hidden_size = 2048
video_embeds = torch.randn(8, hidden_size)
video_grid_thw = torch.tensor([[2, 4, 4]])
second_per_grid_ts = torch.tensor([1.0])

sampling_params = SamplingParams(max_tokens=16)

prompt = {
    "prompt": "<|vision_start|><|video_pad|><|vision_end|>Describe the video.",
    "multi_modal_data": {
        "video": {
            "video_embeds": video_embeds,
            "video_grid_thw": video_grid_thw,
            "second_per_grid_ts": second_per_grid_ts,
        }
    },
}

out = llm.generate([prompt], sampling_params=sampling_params)
print("#####out#####:",out)
```
As Expected:

* Before: crash during EVS processing
* After: completes normally and returns a generation
---

### CODE REVIEW

**Comment by skyloevil** [comment]
@DarkLight1337 @ywang96 PTAL.

**Review by DarkLight1337 — APPROVED** [review]
